### ISSUE TITLE
Is support for SolrCloud on the roadmap?

### ISSUE THREAD

**Issue by mcclurmc** [rsolr/rsolr#107]
We would like to start using SolrCloud, but we can't find a Ruby library that supports it. Are there any plans to integrate SolrCloud support into RSolr? Or is my best bet to use JRuby and SolrJ?

**Comment by m0dd3r** [comment]
What type of 'support' are you looking for? I'm running SolrCloud in production with Sunspot via rsolr as the client. Works great.

**Comment by kmrshntr** [comment]
@mcclurmc We made RSolr connection adapter for SolrCloud. This could enable ruby applications to connect to the Zookeeper ensemble in SolrCloud. Just try it out.
https://github.com/enigmo/rsolr-cloud

**Comment by DaniG2k** [comment]
Indeed, RSolr doesn't seem to work when I start Solr in could mode. Steps to reproduce:

1. Start Solr in cloud mode: `bin/solr start -e cloud`
2. Drop into a Ruby shell and `solr = RSolr.connect url: 'http://127.0.0.1:8983'`
3. This fails: `solr.add {id: 1, title: "test"}`
4. This also fails: `solr.add 'asiagazette', {id: 1, title: "test"}`

I think the first problem is that `.add` doesn't allow me specify a collection name. The second problem is that I'm only connecting to http://127.0.0.1:8983 when by default, Solr also starts up on port `7574`. Not sure if I'm doing this incorrectly...
